.github/dependabot.yml

```diff
@@ -14,3 +14,27 @@ updates:
       - "scylladb/qa-maintainers"
     assignees:
       - "scylladb/qa-maintainers"
+
+  - package-ecosystem: "docker"
```

Comment:
s/docker/docker-loaders/ ?

Reply:
it's a package ecosystem, just to indicate which `package manager` should be used.

[Dependabot Docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem)